### ISSUE TITLE
Results view slow with multiple remote sensing models

### ISSUE THREAD

**Issue by cdbethune** [uncharted-distil/distil#2410]
After we've run more than a handful of searches, things really start to slow down in the results screen when dealing with larger remote sensing models. Specifically, when I ran 20 different searches on a 10K locust set the system became extremely sluggish, to the point where it was not possible to switch models without a wait on the order of 30s.

**Comment by cdbethune** [comment]
duplicate of #2414
